cli/azd/internal/appdetect/java.go

```diff
@@ -203,9 +204,8 @@ func detectDependencies(currentRoot *mavenProject, mavenProject *mavenProject, p
 
 func detectMavenWrapper(path string, executable string) string {
```

Comment:
Hi, @saragluna 

## 1. FYI: `.cmd` is not necessary when run `mvnw` in Windows

Here are screenshots to prove this:

### 1.1. In cmd:

> ![image](https://github.com/user-attachments/assets/673ccf6a-121a-406e-9ae9-fd8b8f23e271)

### 1.2. In Powershell

> ![image](https://github.com/user-attachments/assets/ee443fd5-39c9-464a-ba72-647d547c9445)

## 2. Could you please update related code about `hooks` 

### 2.1. Here is the screenshot about `hooks` related code:

> ![image](https://github.com/user-attachments/assets/320f9fef-f9c0-4537-b107-63436a951ef8)

### 2.2. You can reuse the func in current PR.

1. Get mvn command, no matter it is `mvn` in path or `mvnw` in project.
2. In the code of searching `mvnw`, it will search the folder of current `pom.xml` and all level of parents.
3. You can choose whether to download mvn if both `mvn` and `mvnw` can not be used.

Reply:
In windows, we can use `mvnw` directly instead of using `mvnw.cmd`, as you did in hooks.